build.gradle

```diff
@@ -96,6 +96,14 @@ subprojects {
         include 'NOTICE.txt'   
       }
     }
+    // ignore missing javadocs
+    tasks.withType(Javadoc) { Javadoc javadoc ->
+      // the -quiet here is because of a bug in gradle, in that adding a string option
+      // by itself is not added to the options. By adding quiet, both this option and
+      // the "value" -quiet is added, separated by a space. This is ok since the javadoc
+      // command already adds -quiet, so we are just duplicating it
```

Comment:
Yikes! So it doesn't support adding a flag, only the `-something somevalue` style? And `''` doesn't work for the second parameter? Seems worth of an upstream bug url in the comments.